compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -969,5 +969,11 @@ object SpaceEngine {
 
   def checkMatch(m: Match)(using Context): Unit =
     if exhaustivityCheckable(m.selector) then checkExhaustivity(m)
-    if reachabilityCheckable(m.selector) then checkReachability(m)
+    if reachabilityCheckable(m.selector) then
```

Comment:
We discussed with @noti0na1 and agreed that this change should be removed. It is outdated and should no longer be necessary.